WIP: Comment glyphs

christianalfoni opened the pull request.

lbogdan commented:
<!-- drone -->
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3726/commits/d90016ef9869a5a4649020bb02e5b8688c6ba6d8 is at https://pr3726.build.csb.dev/s/new.

christianalfoni commented:
I did some changes here:

1. I refactored `comments` into its own namespace of Overmind
2. Though the implementation of active glpyhs made perfect sense I moved it way back to where the glyphs are created, so it rather builds everything at the same spot
3. Now you can select code and write a comment and you it actually creates a comment 😮

siddharthkp commented on packages/@types/ot/index.d.ts:
```diff
@@ -21,5 +21,12 @@ declare module 'ot' {
   }
 
-  export { TextOperation };
+  class Selection {
```
t4st comment
